### DISCUSSION TITLE
I/flutter ( 4551): Could not embed because more than one relationship was found for 'friendship' and 'profiles'

### DISCUSSION THREAD

**Discussion by MerlinChiodo** [supabase/supabase#10275] [Questions] [answered]
Hello :),

I'm trying to do a select everything from a friendship table and the username from my profiles table. 

My friendship table looks like this: 
![image](https://user-images.githubusercontent.com/60719496/201425737-4b919405-c14e-42bc-a9bf-bde2e8f9754a.png)

And my profiles table looks like this: 
![image](https://user-images.githubusercontent.com/60719496/201426343-efe59b3f-db70-4a22-977f-9ec435108574.png)

This select throws the error. Probalby a simple solution, or either not possible at all? 
```dart
      _friendships = await supabase
          .from('friendship')
          .select('*, profiles!inner(username)');
```

My optimal result should be complete friendship table and the name of fk_friend1 and the name of fk_friend2

**Comment by GaryAustin1** [comment, marked as answer]
You need to specify each foreign key link by calling it out.  From javascript select reference:
```
const { data, error } = await supabase
  .from('messages')
  .select(`
    content,
    from:sender_id(name),
    to:receiver_id(name)
  `)
  ```
  `from` and `to` are just aliases, so you could do friend1 and friend2.  Then `sender_id` is `fk_friend1`

**Reply by MerlinChiodo** [reply]
Thank you. I did not found the corresponding "from", "to" documentation,
